### DISCUSSION TITLE
how can i fix this

### DISCUSSION THREAD

**Discussion by Joshminemod** [smartcontractkit/full-blockchain-solidity-course-py#841] [Q&A]
contracts/Lottery.sol:4:1: ParserError: Source "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol" not found: File outside of allowed directories.
import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol

**Comment by cromewar** [comment]
Hello @Joshminemod please share your `brownie-config.yaml`.

**Reply by n4n0b1t3** [reply]
@cromewar Yeah, that error message "File outside of allowed directories" is really not helping. It completely throws one of the paths. I would guess that 90% are just typos in the config yaml.
